src/_TextInputWrapper.tsx

```diff
@@ -38,66 +39,55 @@ const sizeVariants = variant({
   }
 })
 
-export type StyledWrapperProps = {
-  disabled?: boolean
-  hasLeadingVisual?: boolean
-  hasTrailingVisual?: boolean
+export type StyledBaseWrapperProps = {
   block?: boolean
   contrast?: boolean
-  validationStatus?: 'error' | 'warning'
-  /** @deprecated Use `size` prop instead */
-  variant?: 'small' | 'large'
-  size?: 'small' | 'large'
+  disabled?: boolean
+  validationStatus?: FormValidationStatus
 } & WidthProps &
   MinWidthProps &
   MaxWidthProps &
   SxProp
 
-export const textInputHorizPadding = '12px'
+export type StyledWrapperProps = {
+  hasLeadingVisual?: boolean
+  hasTrailingVisual?: boolean
+  /** @deprecated Use `size` prop instead */
+  variant?: 'small' | 'large'
+  size?: 'small' | 'large'
+} & StyledBaseWrapperProps
 
-const TextInputWrapper = styled.span<StyledWrapperProps>`
-  min-height: 32px;
+const textInputBasePadding = '12px'
+export const textInputHorizPadding = textInputBasePadding
+
+export const TextInputBaseWrapper = styled.span<StyledBaseWrapperProps>`
```

Comment:
In order to reuse `TextInputWrapper`, I've abstracted the base styles into `TextInputBaseWrapper`, which apply universally to all types of text input, including textarea.